src/pages/DatasetSearch.js

```diff
@@ -0,0 +1,37 @@
+import { useEffect } from 'react';
+import { Notifications } from '../libs/utils';
+import { DataSet } from '../libs/ajax';
+
+export const DatasetSearch = (props) => {
+    useEffect(() => {
+        const init = async () => {
+            const example = {
+                "query": {
+                    "bool": {
+                        "must": [
+                            {
+                                "match": {
+                                    "_type": "dataset"
+                                }
+                            },
+                            {
+                                "match": {
+                                    "_id": 1403
+                                }
+                            }
+                        ]
+                    }
+                }
+            };
+            try {
+                const res = await DataSet.searchDatasetIndex(example);
```

Comment:
I initially created a component for this data, but it didn't seem like it was very useful.